Documentation for Cake for Rider release 0.2.0

nils-a opened the pull request:
Documentation and a very small blog post in preparation of the 0.2.0 release of Cake for Rider.

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -1,4 +1,4 @@
-Order: 30
+Order: 20
```
Why changing order? IntelliSense page has already order 20 (and should be before this page to be consistent with other integration docs)

nils-a replied:
Ah. Because I'm bad at merging things.

nils-a replied:
@pascalberger I fixed that.

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
 
+### Search paths
+
+Allows to modify search paths for building the [Cake tasks window](#running-builds-directly).
```
Something seems wrong in this sentence (people are not building the Cake tasks window)

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
 
 * *Cake file extension*
-  This setting is used to find all Cake files and display them in the tool window.
+  This setting is used to find all Cake files and display them in the Cake tasks window.
```
```suggestion
  This setting is used to find all Cake files and display them in the Cake Tasks window.
```

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -88,15 +88,29 @@ All settings are project - specific and stored in the `.idea` folder. To share s
   Default: `Task\s*?\(\s*?"(.*?)"\s*?\)`
 * *Verbosity*
-  This is the default verbosity to use, when running a task directly from the tool window or when creating a new run configuration.
+  This is the default verbosity to use, when running a task directly from the Cake tasks window or when creating a new run configuration.
```
```suggestion
This is the default verbosity to use, when running a task directly from the Cake Tasks window or when creating a new run configuration.
```

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
 
 The configurations can either be created from an existing Cake task,
-using the [tool window](#cake-tasks-tool-window) or created manually using the run [configuration editor](#editing-run-configurations).
+using the [cake tasks window](#running-builds-directly) or created manually using the run [configuration editor](#editing-run-configurations).
```
```suggestion
using the [Cake Tasks window](#running-builds-directly) or created manually using the run [configuration editor](#editing-run-configurations).
```

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
+## Running builds directly
 
-The Cake tasks tool window lists all Cake scripts and their tasks:
+The Cake tasks window lists all Cake scripts and their tasks:
```
```suggestion
The Cake Tasks window lists all Cake scripts and their tasks:
```

pascalberger commented on input/docs/integrations/editors/rider/run-configurations.md:
```diff
@@ -109,24 +123,35 @@ Default value is `dotnet-cake` and `dotnet-cake.exe` for Windows.
+![Cake task run](/assets/img/cake-rider/docs/cake-run.png){.img-responsive}
 
-Alternatively, the buttons at the top of the tool window can be used to either run the task immediately,
+Alternatively, the buttons at the top of the Cake tasks window can be used to either run the task immediately,
```
```suggestion
Alternatively, the buttons at the top of the Cake Tasks window can be used to either run the task immediately,
or create a new [run configuration](#creating-run-configurations).
```

pascalberger reviewed (changes requested):
Added some minor suggestions

pascalberger reviewed (approved):
LGTM

pascalberger commented:
@nils-a I leave it up to you to merge it once the release is pushed

gep13 reviewed (approved):
LGTM!

nils-a commented:
The release was [approved](https://plugins.jetbrains.com/plugin/15729-cake-for-rider/versions/stable/108104). 
I moved the blog post to "today" and rebased. Will merge as soon as build shows green.
